kolibri/plugins/learn/assets/src/vue/explore-page/index.vue

```diff
@@ -2,10 +2,10 @@
 
   <breadcrumbs v-if="breadcrumbs"></breadcrumbs>
 
-  <card-grid header="Topics" v-if="topics.length">
+  <card-grid header="Topics" v-if="subtopics.length">
     <topic-card
-      v-for="topic in topics"
-      :id="topic.pk"
+      v-for="topic in subtopics"
+      :id="topic.id"
```

Comment:
The API is not returning an `id` field, only `pk`- we would have to modify the serializer if we wanted to do this.